ipaserver/dcerpc.py

```diff
@@ -820,6 +820,8 @@ def __retrieve_trusted_domain_gc_list(self, domain):
 
 
 def string_to_array(what):
+    if isinstance(what, bytes):
```

Comment:
Under Python 2 ``str is bytes``, so this will return a list of strings and not a list of ints.